drivers/spi/Kconfig.mcux_flexio

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2024, STRIM, ALC
+# SPDX-License-Identifier: Apache-2.0
+
+config SPI_MCUX_FLEXIO
+	bool "MCUX FlexIO SPI driver"
```

Comment:
Is this missing `default y`

Reply:
@danieldegrasse, Is it worth making this amendment to this PR?

Reply:
Yes- thank you for adding it